fboss/platform/configs/wedge800cact/platform_manager.json

```diff
@@ -0,0 +1,1659 @@
+{
+  "platformName": "WEDGE800CACT",
+  "rootPmUnitName": "MCB",
+  "rootSlotType": "MCB_SLOT",
+  "slotTypeConfigs": {
+    "MCB_SLOT": {
+      "numOutgoingI2cBuses": 0,
+      "idpromConfig": {
+        "busName": "SMBus I801 adapter at 5000",
+        "address": "0x54",
+        "kernelDeviceName": "24c02"
+      },
+      "pmUnitName": "MCB"
```

Comment:
Please remove the "pmUnitName" names from all slotTypeConfigs.  Since the EEPROM's ProductName field is programmed correctly, this should not be required.

Reply:
> Please remove the "pmUnitName" names from all slotTypeConfigs. Since the EEPROM's ProductName field is programmed correctly, this should not be required.

@somasun 
Could we keep that part? We need pmUnitName when the EEPROM is present but still empty (before the ProductName is programmed). Without pmUnitName, the system cannot bring up the associated CPLD device, and therefore cannot disable the CPLD write-protect. This makes the initial EEPROM provisioning process very difficult.